mlbv/mlbam/mlbgamedata.py

```diff
@@ -53,7 +53,7 @@ def _get_games_by_date(self, date_str=None):
 
         # https://statsapi.mlb.com/api/v1/schedule?sportId=1&startDate=2018-03-26&endDate=2018-03-26&hydrate=schedule.teams,schedule.linescore,schedule.game.content.media.epg
         # hydrate = 'hydrate=schedule.teams,schedule.linescore,schedule.game.content.media.epg'
-        hydrate = 'hydrate=broadcasts(all),game(content(all),editorial(preview,recap)),linescore,team,probablePitcher(note)'
+        hydrate = 'hydrate=broadcasts(all),game(content(media(epg)),editorial(preview,recap)),linescore,team,probablePitcher(note)'
```

Comment:
I haven't tried watching any games yet... What does this change affect? I'm surprised that 'all' isn't pulling in the epg stuff.

Reply:
Okay I'm actually reading the issue now (sorry, I should have brushed up on the issue before commenting)... I see where this is coming from. I'll have to test locally to see if it affects anything. The reason is that I have a couple of local commits as well.

Reply:
Looks good, tested locally. Thanks, and sorry for the wait